djtriggers/tests/test_models.py

```diff
@@ -67,6 +69,36 @@ def test_handle_execution_failure_tries_exceeded(self, mock_logger):
                                                                              try_count=original_tries+1)
         mock_logger.assert_called_with(trigger, message, level=WARNING)
 
+    @override_settings(DJTRIGGERS_TRIES_BEFORE_WARNING=3)
+    @override_settings(DJTRIGGERS_TRIES_BEFORE_ERROR=5)
+    @patch.object(TriggerLogger, 'log_message')
```

Comment:
Any reason why you patched the entire method instead of a few lines?

Reply:
I kept the pattern of the file.

Reply:
I'd still patch more locally unless it's very inconvenient or unreadable. It's in our code conventions, and there's no other practical way to implement code conventions in old code than gradually.